kurtosis-devnet/eigenda.yaml

```diff
@@ -79,8 +79,6 @@ optimism_package:
           - '{{ dig "batcher" "max-l1-tx-size-bytes" 1000 $context }}'
           - --batch-type
           - '{{ dig "batcher" "batch-type" 1 $context }}'
-          - --throttle-threshold=500_000 # start throttling after 500KB of blocks backlog
-          - --throttle-block-size=1000 # throttle op-geth blocks to 1000 gas
```

Comment:
this is what you were talking about that fixed teh tests?

Reply:
These flags are no logger supported. Removed so kurtosis-devnet could start batcher without erroring due to unknown flags passed.

The fix for getting the batcher tests passing ending up being very simple. This change here: https://github.com/Layr-Labs/optimism/pull/52/commits/b21b774dea939054dbdfb5a1fdcc9410688dfed9#diff-6524d10e489e31223b0801a83152fb420341719ae1fb7deab8b74597b5c437e9R224